plugins/zen-seo-lite/includes/class-zen-seo-sitemap.php

```
@@ -135,6 +135,29 @@ private function generate_post_urls()
 
         $posts = \get_posts($args);
 
+        // Batch prime attachment caches to avoid N+1 queries when fetching thumbnails
+        $thumbnail_ids = [];
```

PHPMD (2.15.0)

[error] The variable $thumbnail_ids is not named in camelCase. (undefined)

(CamelCaseVariableName)